hoomd/md/methods/methods.py

```diff
@@ -96,79 +195,56 @@ class NVT(Method):
             :math:`\eta_\mathrm{rot}`)
     """
 
-    def __init__(self, filter, kT, tau):
+    def __init__(self, filter, thermostat=None):
 
         # store metadata
         param_dict = ParameterDict(filter=ParticleFilter,
                                    kT=Variant,
-                                   tau=float(tau),
+                                   thermostat=Thermostat,
```

Comment:
```suggestion
                                   thermostat=OnlyTypes(Thermostat, allow_none=True),
```
This is how we could support `None` on the Python side.